src/pages/instances/actions/CreateImageFromInstanceBtn.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { ActionButton } from "@canonical/react-components";
+import usePortal from "react-useportal";
+import CreateImageFromInstanceForm from "../forms/CreateImageFromInstanceForm";
+
+interface Props {
+  instance: LxdInstance;
+  isDeleting: boolean;
+}
+
+const CreateImageFromInstanceBtn: FC<Props> = ({ instance, isDeleting }) => {
+  const { openPortal, closePortal, isOpen, Portal } = usePortal();
+  const prohibitedStatuses = [
```

Comment:
should use `useInstanceLoading` for `Starting`, `Stopping`, `Freezing` and `Restarting`

Reply:
I am not sure how you wish to implement this..

Reply:
Your implementation is good :)